internal/blackbox_android.py

```diff
@@ -119,18 +119,19 @@ def run_task(self, task):
             if command['command'] == 'navigate':
                 task['page_data']['URL'] = command['target']
                 activity = '{0}/{1}'.format(self.config['package'], self.config['activity'])
-                cmd = 'am start -n {0} -a android.intent.action.VIEW -d "{1}"'.format(activity, \
-                        command['target'].replace('"', '%22'))
+                cmd = 'am start -n {0} -a android.intent.action.VIEW \
+                       -d "{1}"'.format(activity,
+                                        command['target'].replace('"', '%22'))
                 local_intent = os.path.join(task['dir'], 'wpt_intent.sh')
                 remote_intent = '/data/local/tmp/wpt_intent.sh'
                 self.adb.shell(['rm', remote_intent])
                 with open(local_intent, 'wb') as f_out:
                     f_out.write(cmd)
-                if self.adb.adb(['push', local_intent, remote_intent]):
-                    os.remove(local_intent)
-                    self.adb.shell(['chmod', '777', remote_intent])
-                    self.adb.shell([remote_intent])
-                    self.adb.shell(['rm', remote_intent])
+            if self.adb.adb(['push', local_intent, remote_intent]):
```

Comment:
Sorry, I wasn't clear.  It was the "with" but also this "if" block and it's contents also changed indentation.